pkg/minikube/translate/translate_test.go

```diff
@@ -97,3 +102,59 @@ func TestT(t *testing.T) {
 		})
 	}
 }
+
+func TestTranslationFilesValid(t *testing.T) {
+	languages := []string{"de", "es", "fr", "ja", "ko", "pl", "zh-CN"}
```

Comment:
May be instead of hardcoding the list of languages it's better to check all translations/*.json files?
In this case we don't need to update this test if we add a new translation.

Reply:
good idea ! lets do that